doc/documentation_workflow.rst

```diff
@@ -0,0 +1,152 @@
+Documentation workflow
+######################
+
+What you need to know
++++++++++++++++++++++
+
+We use **Sphinx** to generate documentation and **Read the Docs** to publish it.
+
+The documentation for NEST is contained in the ``docs`` directory within the `NEST source code repository <https://github.com/nest/nest-simulator>`_ on GitHub.
+
+Using **GitHub** allows the NEST simulator documentation to live alongside its code. Our developers benefit from smooth version control, with each release of NEST having its own documentation.
+
+.. image:: _static/img/documentation_workflow.png
+  :width: 300
+  :alt: Alternative text
+
+Changing the documentation
+++++++++++++++++++++++++++
+
+If you notice something out of place, you can submit an `issue <https://github.com/nest/nest-simulator/issues>`_ in our GitHub repository.
+
+You can also make changes directly to your forked copy of the `NEST source code repository <https://github.com/nest/nest-simulator>`_ and create a `pull request <https://github.com/nest/nest-simulator/pulls>`_. Just follow the workflow below!
+
+Setting up your environment
++++++++++++++++++++++++++++
+
+To keep things simple, we have created a conda environment for you. Installing it will enable you to smoothly generate documentation for NEST.
+
+If you are using Linux and want to install a full development environment:
+
+1. Install conda (we recommend `miniconda <https://docs.conda.io/en/latest/miniconda.html#>`_).
+
+2. Switch to the ``doc`` folder in the source directory:
+
+.. code-block:: bash
+
+    cd git/nest-simulator/doc
+
+3. Create and activate the environment:
+
+.. code-block:: bash
+
+   conda env create -f conda_environment.yml
+   conda update -n base -c defaults conda
+   conda activate doc
```

Comment:
In #1411 I renamed the environment to `nest-doc`. Now it should be:

   ` conda activate nest-doc`

Reply:
Done.